src/ioctls/vm.rs

```diff
@@ -1584,6 +1612,14 @@ mod tests {
         assert!(vm.set_tss_address(0xfffb_d000).is_ok());
     }
 
+    #[test]
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    fn test_set_identity_map_address() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        assert!(vm.set_identity_map_address(0xfffb_c000).is_ok());
```

Comment:
I see the unit test is failing. Maybe you need to check for the capability in case the test system kernel is too old?

Reply:
Yes I added this check, but the failure was about an error in the implementation of the ioctl. It's all fixed now.